### ISSUE TITLE
OBO loader creates weird CVs when id has a colon in OBO

### ISSUE THREAD

**Issue by bradfordcondon** [tripal/tripal#525]
## BUG/ERROR report
https://www.hardwoodgenomics.org/cv/lookup/
note the CVs for "fatty acid 18" and "fatty acid anion 18" 
https://www.hardwoodgenomics.org/cv/lookup/fatty%20acid%20anion%2018

Other than these two terms, the Plant trait ontology loads fine. (really awesome job with the cross-ontology lookup)

It's clear why this is happening: look at the fatty acid lines in the PTO:

`id: CHEBI:132502 ! fatty acid 18:3`  it's splitting on the second : for the DB shortname

### System information

* Tripal Version: latest master, tripal 3
#### Steps to reproduce
Load the plant trait ontology at http://purl.obolibrary.org/obo/to.obo
visit the CV lookup area of your site

<img width="1440" alt="screen shot 2018-07-31 at 6 14 52 pm" src="https://user-images.githubusercontent.com/7063154/43490192-ad904ccc-94ed-11e8-961d-89f7534f9fb2.png">

**Comment by shawnawsu** [comment]
I'm just testing out an update for a fix for this, it's just taking ages to test on my local.

**Comment by bradfordcondon** [comment]
i know developing for this must have been the worst - as i tried to note, its HUGELY improved, so thank you!

**Comment by shawnawsu** [comment]
That's great to hear! I have to admit the 'rules' that aren't really rules are frustrating but we're getting to a good place.

**Comment by bradfordcondon** [comment]
here is a mini TO file to make testing this issue easier

```
format-version: 1.2
data-version: releases/2018-03-09
saved-by: cooperl
default-namespace: plant_trait_ontology
import: http://purl.obolibrary.org/obo/po.owl
import: http://purl.obolibrary.org/obo/to/imports/chebi_import.owl

ontology: to

[Term]
id: CHEBI:132502 ! fatty acid 18:3
is_a: CHEBI:73155 ! trienoic fatty acid

[Term]
id: CHEBI:73155 ! trienoic fatty acid
```


edit: oops this doesnt work, sorry haha

**Comment by bradfordcondon** [comment]
this is resolved in #627
